test/Dataworker.loadData.ts

```diff
@@ -343,6 +359,69 @@ describe("Dataworker: Load data used in all functions", async function () {
           .div(fixedPointAdjustment),
       });
     });
+
+    it("Saves V3 fast fill under correct repayment chain and repayment token when dealing with lite chains", async function () {
+      // Mock the config store client to include the lite chain index.
+      mockConfigStore.updateGlobalConfig(
+        GLOBAL_CONFIG_STORE_KEYS.LITE_CHAIN_ID_INDICES,
+        JSON.stringify([spokePoolClient_1.chainId])
+      );
+      await mockConfigStore.update();
+      // Ensure that our test has the right setup.
+      expect(mockConfigStore.liteChainIndicesUpdates.length).to.equal(1);
+      mockConfigStore.liteChainIndicesUpdates[0].timestamp = 0;
+      expect(repaymentChainId).to.not.eq(spokePoolClient_1.chainId);
```

Comment:
```suggestion
      expect(repaymentChainId).to.not.eq(originChainId);
```

is this more correct for preserving this test's usefulness?

Reply:
https://github.com/across-protocol/relayer/pull/1629/commits/a489074b3e00abe73d27929ddb2c69b953ad86b4